client/components/data/query-contact-details-cache/index.jsx

```diff
@@ -24,11 +24,11 @@ class QueryContactDetailsCache extends Component {
 }
 
 QueryContactDetailsCache.propTypes = {
-	requesting: PropTypes.bool,
-	requestContactDetailsCache: PropTypes.func
+	isRequesting: PropTypes.bool.isRequired,
```

Comment:
... yeah, I think I'm basically wrong. You can fail to pass `isRequesting` once and undefined is the basically the same as false, but if you keep failing to pass it in, you're going to generate another network request every time the component mounts, which is definitely not what you really want.